pkg/controller/volume/attachdetach/cache/actual_state_of_world.go

```diff
@@ -254,6 +255,7 @@ func (asw *actualStateOfWorld) AddVolumeToReportAsAttached(
 	asw.addVolumeToReportAsAttached(volumeName, nodeName)
 }
 
+// TODO: I note devicePath is an arg.  Are we calling this only when the volume is fully attached I wonder?  Is this only called on the kubelet side?  Is this called at all?
```

Comment:
This function is called on master by reconciler after it successfully attach a volume to a node.